.gitignore

```diff
@@ -54,6 +54,7 @@ playground.xcworkspace
 # Carthage/Checkouts
 
 Carthage/Build
```

Comment:
Generic question for you for all our projects: Why don't we ignore Carthage, the checkouts only add to the size of the remote repo.

Reply:
For this project it doesn't really matter as we don't have any dependencies currently... but I wasn't actually aware this was the case! Definitely something we need to look into 👀

Reply:
Sure :) No biggie anyway